roomserver/storage/interface.go

```diff
@@ -79,7 +79,7 @@ type Database interface {
 	// Look up the state entries for a list of string event IDs
 	// Returns an error if the there is an error talking to the database
 	// Returns a types.MissingEventError if the event IDs aren't in the database.
```

Comment:
Need to comment how `excludeRejected` interacts with this. If all `eventIDs` are excluded, does it return `[]` or a `types.MissingEventError`?

Reply:
Updated the comment, it returns `[]`.